.github/workflows/linux.yml

```diff
@@ -30,8 +30,28 @@ jobs:
     - name: Lint the code
       uses: ./.github/actions/lint
 
+  GetMatrix:
+    runs-on: ubuntu-latest
+
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+
+    steps:
+    - name: Determine job matrix
+      id: set-matrix
+      run: |
+        set -euo pipefail
+
+        MATRIX='{"build-type": ["Debug", "Clang", "Release"]}'
```

Comment:
Nope, you need to spell out `{"build-type": "Debug"}`, multiple times, see for example:

https://github.com/canonical/multipass-blueprints/actions/runs/2256642519/workflow#L36-L38